board/m2k/post-build.sh

```diff
@@ -32,12 +32,12 @@ genimage                           \
 	--outputpath "${TARGET_DIR}/opt/" \
 	--config "${GENIMAGE_CFG}"
 
-rm ${TARGET_DIR}/opt/boot.vfat
-rm ${TARGET_DIR}/etc/init.d/S99iiod
+rm -f ${TARGET_DIR}/opt/boot.vfat
+rm -f ${TARGET_DIR}/etc/init.d/S99iiod
 
-mkdir -p ${TARGET_DIR}/www/img
-mkdir -p ${TARGET_DIR}/mnt_jffs2
-mkdir -p ${TARGET_DIR}/etc/wpa_supplicant/
+[ -d "${TARGET_DIR}/www/img" ] || mkdir -p ${TARGET_DIR}/www/img
```

Comment:
`mkdir -p ...` will return 0 even if the folder already exists, so this check is redundant.

Reply:
good point ; will update